frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/activity-card/ocs-upgrade-activity.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import ActivityItem from '@console/shared/src/components/dashboard/activity-card/ActivityItem';
+import { SubscriptionState } from '@console/operator-lifecycle-manager';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const isOCSUpgradeActivity = (subscription: K8sResourceKind): boolean => {
```

Comment:
can be one liner and should reuse `getSubscriptionStatus` from `operator-lifecycle` plugin
```suggestion
export const isOCSUpgradeActivity = (subscription: SubscriptionKind): boolean => getSubscriptionStatus(subscription) === SubscriptionState.SubscriptionStateUpgradePending;
```